### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by mmattel**
This is an improvement of README.md based on a discussion with @settermjd 
There will be a another commit in docs-ui to improve that readme .

### CODE REVIEW

File: docs/test-ui-bundle.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Testing unmerged docs-ui changes
+
+If you want to test an unmerged change made in the `docs-ui` reprository with full content, build as usual the
```
Comment (settermjd) [inline docs/test-ui-bundle.md]: I suggest changing "with full content" to "with content from the ownCloud documentation".

File: docs/test-ui-bundle.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+If you want to test an unmerged change made in the `docs-ui` reprository with full content, build as usual the 
+documentation with the antora command but add the option `--ui-bundle-url <path-to-your-local-ui-bundle>/ui-bundle.zip` to your command.
+The location for this bundle is in the `docs-ui`reprository in directory `build/`.
+More details how the create the `ui-bundle.zip` can be found in the `docs-ui`reprository.
+
+The following example assumes:
+
+- You are in the root directory of your local `docs`repository
+- The `docs-ui`repository directory is on the same directory level as `docs`
+- You have `ui-bundle.zip` manually created with `gulp pack` (for details see `docs-ui`)
```
Comment (settermjd) [inline docs/test-ui-bundle.md]: Why not just link "with `gulp pack`" to the relevant section of the documentation (and remove the rest of the sentence)?

File: docs/test-ui-bundle.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+```
+antora --clean \
+       --pull \
+       --ui-bundle-url ../docs/build/ui-bundle.zip \
+       --url http://localhost:5000 \
+       site.yml
+```
+
+If the build returns without an error, open the documentation with your browser to see your changes.
```
Comment (settermjd) [inline docs/test-ui-bundle.md]: How about linking "open the documentation with your browser" to the relevant section of the documentation that shows how to start a locally running copy?

Comment (mmattel) [reply docs/test-ui-bundle.md]: We do not have a "one shot fits all" section for that as there are multiple posibilities. Just see the broken link checker stuff. That is the reason how I wrote it as I did including a item in the assumption list.

Comment (settermjd) [reply docs/test-ui-bundle.md]: To me, it's not completely clear, especially for newcomers.

File: docs/test-ui-bundle.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Testing unmerged docs-ui changes
+
+If you want to test an unmerged change made in the `docs-ui` reprository with content from the ownCloud documentation, 
+build as usual the documentation with the antora command but add the option 
+`--ui-bundle-url <path-to-your-local-ui-bundle>/ui-bundle.zip` to your command.
+The location for this bundle is in the `docs-ui`reprository in directory `build/`.
```
Comment (tboerger) [inline docs/test-ui-bundle.md]: Did you fix the other whitespaces? Or was it a rendering issue? But here is still one missing.

Comment (mmattel) [reply docs/test-ui-bundle.md]: There was really one missing...

**Pull request merged**